README.md

```diff
@@ -20,12 +20,13 @@ The OpenTrace reference implementation comprises:
 
 
 ## Prerequisites:
-+ [Node.js](https://nodejs.org/)
++ [Node.js](https://nodejs.org/) v10
 
 
 ## Create Firebase Project
 1. Create a new Firebase Project from [Firebase console](https://console.firebase.google.com/).
 2. Enable Google Analytics for the project, to be used for Firebase Crashlytics and Firebase Remote Config.
+3. Make sure to upgrade the project from the "Spark" free plan to the "Blaze" pay-as-you-go plan to avoid future quota issues.
```

Comment:
Not too sure if this is necessary as the responsibility for managing quota is on the user, regardless of what SaaS offering they're using.